spec/predicates/simple-verification.md

```diff
@@ -0,0 +1,108 @@
+# Predicate type: Simple Verification
+
+Type URI: https://in-toto.io/attestation/simple-verification/v0.1
+
+Version: 0.1
+
+Authors: Tom Hennen (@TomHennen), Andrew McNamara (@arewm)
+
+## Purpose
+
+A Simple Verification Attestation (SVA) communicates that an artifact has been verified against one or more policies, and records the properties that were verified. This enables consumers to trust a concise summary of verification results, without requiring access to all underlying attestations or policies.
+
+## Use Cases
+
+-   Allowing software consumers to delegate complex policy decisions to a trusted verifier, and rely on a simple summary of which properties were verified.
+-   Supporting evolving verification needs (e.g., SLSA, vulnerability scanning, organizational policies) without coupling the SVA format to any specific framework.
+-   Enabling verifiers to add additional information (e.g., policy references, expiration, property-level timing) as needed, while maintaining a simple core schema.
+
+## Prerequisites
+
+This predicate depends on the [in-toto Attestation Framework]. Familiarity with the concept of attestation statements and the use of URIs for identifying verifiers and resources is assumed.
+
+## Model
+
+An SVA is an attestation that a verifier (`verifier.id`) has checked an artifact (`resourceUri`) at a specific time (`timeVerified`) and asserts a set of verified properties (`properties`). The SVA does not include information for reproducing the verification result. Instead, verifiers MAY generate additional provenance attestations for the simple verification attestation as needed.
+
+## Schema
+
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/simple-verification/v0.1",
+  "predicate": {
+    "verifier": {
+      "id": "<URI>"
+    },
```

Comment:
VSAs supported additional verifier metadata with verifier versions. Was this found to not be necessary, or if we wanted to include such metadata, would we do so as a property now?

Reply:
When @TomHennen and I were discussing this, we were talking about how the additional information was often confusing. This can certainly still be captured in a provenance attestation for the SVA itself.

Reply:
@haydentherapper , do you think that we should provide more information on this in the predicate specification?

Reply:
Thinking more, does a consumer of a SVA need to know that a verifier has changed its verification logic? These verifiers aren't expected to be auditable, so a new revision of a verifier isn't meaningful, unless we expect that verifiers will publish the list of properties that they may include in a SVA.

Another option is to ask that verifiers version their identifiers, as in if they update their verification logic, they use a new unique ID. This puts additional work on the SVA consumer however, to maintain an up to date list of verifier IDs.

Reply:
If a verifier uses a policy which can change the logic, an extra field can be used to indicate what the policy is. This would be optional and could be used by consumers if they knew about this implementation detail. I think it makes sense for the verifier to be versioned though.